server/src/routes/classroom.ts

```diff
@@ -0,0 +1,12 @@
+import classroomController from "../controllers/classroom";
+import { Router } from "express";
+
+const router = Router();
+
+router.post("/", (req, resp) => classroomController.create(req, resp));
+router.get("/", (req, resp) => classroomController.get(req, resp));
+router.get("/list", (req, resp) => classroomController.getAll(req, resp));
```

Comment:
Per REST, the URL should just be `/`. The others should be like the comment in a previous PR.